pywinauto/controls/common_controls.py

```diff
@@ -657,6 +657,36 @@ def deselect(self):
     # Non PEP-8 alias
     Deselect = deselect
 
+    #-----------------------------------------------------------
+    def search_editor(self, friendly_class_name=""):
+        """Return the editor HwndWrapper of the item
+
+        Possible ``friendly_class_name`` values:
+
+        * ``""``  Returns most possible editor of the item
+        * ``"friendlyclassname"``  Returns editor with particular friendlyclassname
+        """
+        # If currently editing in this item or some other
+        self.listview_ctrl.type_keys("{ENTER}")

```

Comment:
Probably we need some small pause here while in-place ctrl initialization is done. Or timing inside `type_keys` is enough? I'm not sure. If you decide it, please add new timing in `timings.Timings` object.